python/lsst/pipe/analysis/utils.py

```diff
@@ -712,8 +715,8 @@ def calibrateSourceCatalog(catalog, zp):
     # Convert to constant zero point, as for the coadds
     fluxKeys, errKeys = getFluxKeys(catalog.schema)
     factor = 10.0**(0.4*zp)
-    for name, key in fluxKeys.items() + errKeys.items():
-        catalog[key] /= factor
+    for name, key in list(fluxKeys.items()) + list(errKeys.items()):
+       catalog[key] /= factor
```

Comment:
Why did this line move?

Reply:
Your eyes on this would be great @timj as I am not really a qualified reviewer for this one (I'm only vaguely aware of some of the Py3 required changes).  Are you just poking, or would you be willing to give it the official review?

Reply:
I'm fine being the "official" reviewer (I did approve the PR) in the sense that the changes look fine to me. I haven't tried it on python3 though and I'm not sure if some of these changes were found by iteratively debugging or by using futurize.

Reply:
Great, thanks!  I made the change on Jira.

If Sophie says things are running and producing plots on python3, that's good enough confirmation for me :)